core-bundle/src/Resources/contao/library/Contao/Controller.php

```diff
@@ -968,6 +980,24 @@ public static function replaceDynamicScriptTags($strBuffer)
 					}
 				}
 			}
+
+			if ($objCombinerDefer->hasEntries())
+			{
+				if ($blnCombineScripts)
+				{
+					$strScripts = Template::generateScriptTag($objCombinerDefer->getCombinedFile(), true) . $strScripts;
+				}
+				else
+				{
+					$arrReversed = array_reverse($objCombinerDefer->getFileUrls());
+
+					foreach ($arrReversed as $strUrl)
+					{
+						$options = StringUtil::resolveFlaggedUrl($strUrl);
+						$strScripts = Template::generateScriptTag($strUrl, false, $options->mtime, null, null, null, true) . $strScripts;
+					}
+				}
+			}
```

Comment:
Shouldn't this come directly after the `if ($objCombinerAsync->hasEntries())` block, within the same `if (!empty($GLOBALS['TL_JAVASCRIPT']) && \is_array($GLOBALS['TL_JAVASCRIPT']))` block?

Reply:
It is. Github displays this a bit weird.

Reply:
Sorry, should have checked in the orignal source file :)